ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -95,14 +68,11 @@ export default class ClientsCountsRoute extends Route {
     const endTimestamp = Number(params.end_time) || getUnixTime(timestamp.now());
     const { activity, activityError } = await this.getActivity(startTimestamp, endTimestamp);
 
-    const isSecretsSyncActivated = await this.isSecretsSyncActivated(activity);
-
     return {
       activity,
       activityError,
       config,
       endTimestamp,
-      isSecretsSyncActivated,
```

Comment:
Great idea moving this to the `counts.ts` component! Much clearer there than passing down from this route model hook 🥇